### PR TITLE
Note removal of timer job in SharePoint 2019

### PR DESCRIPTION
**Pull request by TroyStarr**
The "Document and List Static Data Fixed Sample job" timer job is removed from SharePoint Server 2019 by the March 2021 Public Update for SharePoint Server 2019 (KB 4493230). Updating the documentation with this information.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/technical-reference/default-timer-jobs-in-sharepoint-server-2019.md [hunk 1]
```diff
@@ -243,6 +243,8 @@ The following table lists the default timer jobs for SharePoint Server 2019.
...
 | Workflow Failover                                                                                                                          | Processes events for workflows that have failed and are marked to be retried.                                                                                                                                                                                                                                                       | 15 Minutes           |
 
+<a name="Note1"><sup>1</sup></a> This timer job is not needed by SharePoint Server 2019 and is removed by the <a href="https://support.microsoft.com/topic/description-of-the-security-update-for-sharepoint-server-2019-march-9-2021-kb4493230-123bf257-222a-41f5-2ad4-96efe8b5df87">Security Update for SharePoint Server 2019: March 9, 2021 (KB4493230)</a>.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/technical-reference/default-timer-jobs-in-sharepoint-server-2019.md:246]:
```suggestion
<a name="Note1"><sup>1</sup></a> This timer job is not needed by SharePoint Server 2019 and is removed by the <a href="https://support.microsoft.com/topic/description-of-the-security-update-for-sharepoint-server-2019-march-9-2021-kb4493230-123bf257-222a-41f5-2ad4-96efe8b5df87">security update for SharePoint Server 2019: March 9, 2021 (KB4493230)</a>.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@TroyStarr hope you are well. Wondering if the changes that Johan suggested can be made? Thanks

**Comment by TroyStarr** [comment]
Hi @scanum, yeah, Johan's changes look good to me.  Feel free to proceed with them.  Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**